plotst/axis.typ

```diff
@@ -143,10 +145,21 @@
       pos.at(1) = -length + pos.at(1)
       invert_markings = -1
     }
-    
+    let arrow_size = 0pt
```

Comment:
Could this be a controllable parameter? Maybe setting to 0pt to disable.

Reply:
I also thought about that but then I came to the conclusion that too much control over how the graph looks like might be overkill. As an example you also don't really want to be able to configure the length axis markings.

Reply:
Ok fine, but using the old import method is always a bit clunky